src/haz3lweb/view/TestView.re

```diff
@@ -88,7 +90,7 @@ let test_bar_segment = (~inject, (_id, reports)) => {
     ~attr=
       Attr.many([
         clss(["segment", status]),
-        Attr.on_click(jump_to_test(~inject)),
+        Attr.on_click(jump_to_test(~inject, _id)),
```

Comment:
variables with leading underscores indicate an unused variable, so this can be renamed to `id` in both places